abs.yaml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - sfe1ed40
+  - https://staging.continuum.io/prefect/fs/dbt-semantic-interfaces-feedstock/pr1/cd87abc
```

Comment:
I will remove thse staging URLs as soon as the packages are available on sfe1ed40.